cdap-ui/app/cdap/components/AppDetailedView/index.js

```diff
@@ -162,12 +162,6 @@ export default class AppDetailedView extends Component {
     }
     this.setState({
       successMessage
-    }, () => {
-      setTimeout(() => {
-        this.setState({
-          successMessage: null
-        });
-      }, 3000);
```

Comment:
If this is removed, the success message will always persist?

Reply:
No. The `OverviewHeader` component has a timeout function that automatically closes the success message. This was redundant.